app/src/main/java/com/keylesspalace/tusky/fragment/ViewVideoFragment.kt

```diff
@@ -395,18 +422,24 @@ class ViewVideoFragment : ViewMediaFragment(), Injectable {
             })
             .start()
 
-        if (visible && (binding.videoView.player?.isPlaying == true) && !isAudio) {
-            hideToolbarAfterDelay(TOOLBAR_HIDE_DELAY_MS)
```

Comment:
Because there is a `hideToolbarAfterDelay `in the `onIsPlayingChanged`, we ALWAYS get an `onIsPlayingChanged`, and `onIsPlayingChanged` always happens last, we do not need any of the other `hideToolbarAfterDelay`s.